pyannote/audio/features/pretrained.py

```diff
@@ -136,7 +136,8 @@ def __init__(
         self.device = torch.device(device)
 
         # send model to device
-        self.model_ = model.eval().to(self.device)
+        with torch.no_grad():
+            self.model_ = model.eval().to(self.device)
```

Comment:
I think this does nothing and can be safely removed.
@MarvinLvn  can you double check?

Reply:
You're right ! My bad. I just deleted it.